src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -30,62 +30,73 @@ namespace Microsoft.Azure.Commands.Sql.FailoverGroup.Cmdlet
         [ValidateNotNullOrEmpty]
         public string PartnerServerName { get; set; }
 
         /// <summary>
         /// Gets or sets the failover policy without data loss for the Sql Azure Failover Group.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The failover policy without data loss for the failover group.")]
+            HelpMessage = "The failover policy of the Azure SQL Database Failover Group.")]
         [ValidateNotNullOrEmpty]
         public FailoverPolicy FailoverPolicy { get; set; }
 
         /// <summary>
         /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
         [Parameter(Mandatory = false,
-            HelpMessage = "The window of grace period that we tolerate with data loss during a failover operation for the failover group.")]
+            HelpMessage = "The grace period during outage before automatic failover with data loss of the Failover Group is triggered.")]
         [ValidateNotNullOrEmpty]
-        [Obsolete("This parameter will be deprecated in the next release.")]
-        public int GracePeriodWithDataLossHour { get; set; }
+        [ValidateRange(0, int.MaxValue)]
+        public int GracePeriodWithDataLossHours { get; set; }
 
         /// <summary>
-        /// Gets or sets the failover policy for read only endpoint of theSql Azure Failover Group.
+        /// Gets or sets the grace period with data loss for the Sql Azure Failover Group.
         /// </summary>
-        [Parameter(Mandatory = false,
-            HelpMessage = "The failover policy for read only endpoint of the failover group.")]
+        [Parameter(Mandatory = false, DontShow = true,
+            HelpMessage = "The grace period during outage before automatic failover with data loss of the Failover Group is triggered.")]
         [ValidateNotNullOrEmpty]
-        public AllowReadOnlyFailoverToPrimary AllowReadOnlyFailoverToPrimary { get; set; }
+        [ValidateRange(0, int.MaxValue)]
+        [Obsolete("This parameter will be deprecated in the next release.")]
+        public int GracePeriodWithDataLossHour { get; set; }
 
         /// <summary>
-        /// Gets or sets the tag associated with the Azure SQL Database Failover Group
+        /// Gets or sets the failover policy for read only endpoint of the Sql Azure Failover Group.
         /// </summary>
-        /// 
-        [Obsolete("This parameter will be deprecated in the next release.")]
-        [Parameter(Mandatory = false,
-            HelpMessage = "The tag to associate with the Azure SQL Database Failover Group")]
-        public Hashtable Tag { get; set; }
+        [Parameter(Mandatory = false, DontShow = true,
+            HelpMessage = "Whether an outage on the secondary server should trigger automatic failover of the read-only endpoint. This feature is not yet supported.")]
```

Comment:
The parameter is already added and cannot be removed since it is not obsoleted. The github diff doesn't seem to be doing a good job of clearly showing the changes.